src/sample.mzml

```diff
@@ -0,0 +1,5000 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
**issue:** Remove file from git.

I think it's good to have some simple unit tests to verify that file reading works correctly. Rather than adding example files to git, they can be simulated on the fly in the unit tests. [Here's](https://github.com/Noble-Lab/casanovo/blob/scan_id/tests/conftest.py) an example.

Reply:
Added unit tests to for mgf and mzml query read methods.

Reply:
**todo:** This file still needs to be removed from the version control.